src/annotation.ts

```diff
@@ -149,23 +186,20 @@ export class Annotation extends Modifier {
   draw(): void {
     const ctx = this.checkContext();
     const note = this.checkAttachedNote();
-    this.setRendered();
-
+    const stemDirection = note.hasStem() ? note.getStemDirection() : Stem.UP;
+    const textFormatter = TextFormatter.create(this.textFont);
     const start = note.getModifierStartXY(ModifierPosition.ABOVE, this.index);
 
+    this.setRendered();
+
     // We're changing context parameters. Save current state.
     ctx.save();
     const classString = Object.keys(this.getAttribute('classes')).join(' ');
     ctx.openGroup(classString, this.getAttribute('id'));
     ctx.setFont(this.textFont);
 
     const text_width = ctx.measureText(this.text).width;
-
-    // Estimate text height to be the same as the width of an 'm'.
-    //
-    // This is a hack to work around the inability to measure text height
-    // in HTML5 Canvas (and SVG).
-    const text_height = ctx.measureText('m').width;
+    const text_height = textFormatter.maxHeight + 2;
```

Comment:
Why not `ctx.measureText(this.text).height` rather than `ctx.measureText(this.text).width`?
Please note than in #1262 I am trying to fix the implementation of `ctx.measureText(this.text).height`

Reply:
You usually want max height and actual width with text, for placement.  Because text has a baseline.  you might want the actual height in some cases, like if you wanted to draw a glyph or a box around it.

Reply:
Not sure what's up with the tabs.  It looks like there was some behavior specific to tab annotations.  I'll check on it.

Reply:
> You usually want max height and actual width with text, for placement. Because text has a baseline. you might want the actual height in some cases, like if you wanted to draw a glyph or a box around it.

What do you think that we should do here? Should `ctx.measureText(this.text).height` return the actual height of `this.text` or `textFormatter.maxHeight`

Reply:
>> ctx.measureText(this.text).height return the actual height 

I think so.  Probably the most useful thing would be to return a structure with width, height, maxHeight, and possibly advanceWidth.  90+% of the time, people only care about the width, anyway.

Reply:
In the end I standarised  in #1262 `measureText` to the svg behaviour. I suggest to change:
```
 const text_width = ctx.measureText(this.text).width;
 const text_height = textFormatter.maxHeight + 2;
```
to
```
const textMeasure = ctx.measureText(this.text);
const text_width = textMeasure.width;
const text_height = textMeasure.height + 2;
```